### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by andriyParashchuk**
https://freereports.rigor.com/index.html?tix=eaa8fd116342993cbd9c9033c75afed7

Staging - http://447-performance-stage.surge.sh/

### CODE REVIEW

**Comment by andriyParashchuk** [comment]
Before:
image

After:
image

**Comment by andriyParashchuk** [comment]
Please review

File: .htaccess [hunk 1]
```diff
@@ -0,0 +1 @@
+Cache-Control: max-age=604800
```
Comment (pftg) [inline .htaccess]: will github use this file?

Comment (pftg) [reply .htaccess]: yep, github does not support this. Please put this into HTML meta tag, then maybe browser can use it

Comment (pftg) [reply .htaccess]: https://www.metatags.org/all-meta-tags-overview/meta-equiv-cache-control/

Comment (andriyParashchuk) [reply .htaccess]: Yes, you are right

**Comment by pftg** [comment]
Deployed to https://jetthoughts-pr-447.surge.sh

**Comment by dgorodnichy** [comment]
LGTM

**Pull request merged**